src/popup/components/TransactionOverview.vue

```diff
@@ -136,8 +138,23 @@ export default defineComponent({
             },
             title: transactionTypes[txType.value],
           };
+        case TX_TYPE_MDW.PayingForTx: {
```

Comment:
Actually `PayingForTx` is a wrapper on top of a transaction so, the inner transaction should be on top of this switch, because our plan is just to add additional details row to a properly parsed inner transaction.